spec/support/shared_contexts/stub_previously_funding_check_for_all_courses.rb

```diff
@@ -1,13 +1,12 @@
 RSpec.shared_context("Stub previously funding check for all courses") do # rubocop:disable RSpec/ContextWording:
-  let(:api_call_get_an_identity_id) { raise NotImplementedError }
   let(:api_call_trn) { raise NotImplementedError }
 
   before do
     Course.pluck(:identifier).each do |course_identifier|
       mock_previous_funding_api_request(
         course_identifier:,
         trn: api_call_trn,
-        get_an_identity_id: api_call_get_an_identity_id,
+        get_an_identity_id: user_uid,
```

Comment:
Is user_uid always going to be set where this is used?

Reply:
Yeah, it's set as a part of "Stub Get An Identity Omniauth Responses" which is required for all flows now since all users hit the get an identity flow.